content/resources/cheatsheet.md

```diff
@@ -94,17 +94,19 @@ figure_list:
 
 ## Shortcodes
 
-A shortcode is a snippet of code inserted in a content (`.md`) file that pulls in information from other files in your project. For example, in Quire, the `q-figure` shortcode is used to insert an image from the `static/img` folder. It additionally references the `id`, `caption`, and other optional information from your `figures.yaml` data file. Other shortcodes include `q-cite`, `q-bibliography`, and `q-contributor.` Examples of proper formatting for these shortcodes are below.
+A shortcode is a snippet of code inserted in a content (`.md`) file that pulls in information from other files in your project. For example, in Quire, the `% figure %` shortcode is used to insert an image from the `content/_assets/images` folder. It additionally references the `id`, `caption`, and other optional information from your `figures.yaml` data file. Other shortcodes include `% cite %`, `% contributors %`, and `% accordion %`. Examples of proper formatting for these shortcodes are below.
```

Comment:
Please correct shortcode references to {% figure %}, {% cite %}, etc. Thank you!